### DISCUSSION TITLE
ccache.log size up to 15G

### DISCUSSION THREAD

**Discussion by sunmaxwell** [ccache/ccache#441] [Support]
### Question ###

In my project, ccache.log size up to scaring 15G!   when opened , I got lots of information like the following:

`127293694 [2019-07-08T15:46:26.968759 107670] Include file ../../../../../../../usr/include/c++/5/tuple ctime too new
127293695 [2019-07-08T15:46:26.968795 107670] Include file ../../../../../../../usr/include/c++/5/functional ctime too new`

Unfortunately, I know little about why this informative log is generated and how to handle with it.
Great thanks for your help!

**Comment by afbjorklund** [comment]
The ccache log is a verbose debugging log, and should only be enabled while troubleshooting.

I don't think there was anything particularly wrong with that line, it was just being informative...

**Comment by sunmaxwell** [comment]
> The ccache log is a verbose debugging log, and should only be enabled while troubleshooting.
> 
> I don't think there was anything particularly wrong with that line, it was just being informative...

sorry about my description! I've corrected it. And finally find that it is a tool that I use caused this issue.That is a tool to stub time function. Thanks afbjorklund!

**Comment by afbjorklund** [comment]
We had some discussions about a less verbose ccache log, that could be enabled always (but it is not implemented). Also, when setting up any kind of logging output you should take a look at logrotate(8)
